llzk-sys/src/sanity_tests/builder.rs

```diff
@@ -38,5 +39,6 @@ fn test_mlir_op_builder_listener_create() {
     }
 }
 
-unsafe extern "C" fn test_callback1(_: MlirOperation, _: *mut c_void) {}
-unsafe extern "C" fn test_callback2(_: MlirBlock, _: *mut c_void) {}
+unsafe extern "C" fn test_callback1(_: MlirOperation, _: MlirOpBuilderInsertPoint, _: *mut c_void) {
```

Comment:
What are these callbacks supposed to do and the main differences between them?

Reply:
They are listeners for the OpBuilder. In this context they are empty functions whose only purpose is to exists since I cannot have a function pointer to null. Rust won't allow it.

Reply:
Add a comment explaining that please.